Frontend/ui-common/src/Engineering/Helpers/Log.js

```diff
@@ -28,3 +28,7 @@ export const logDebug = function (level) {
     }
   };
 };
+
+export const getWindowKey = function (key) {
+  return window[key] ? window[key] : "";
```

Comment:
@ashishsingh101  we already have a function to extract the value from window : `getAppConfigFromWindow`. Can we use the same for this purpose?

Reply:
using that function is causing cycling dependency